implementations.md

```diff
@@ -0,0 +1,11 @@
+# Implementations of Container Specification
+
+## Runtime
+
+* [opencontainers/runc](https://github.com/opencontainers/runc) - reference implementation of OCI runtime
+* [hyperhq/runv](https://github.com/hyperhq/runv) - Hypervisor-based runtime for OCI
+
+## Testing
+
+* [huawei-openlab/ocp-testing](https://github.com/huawei-openlab/ocp-testing) - Testing framework for OCI configuration and runtime
```

Comment:
On Tue, Aug 04, 2015 at 11:14:27AM -0700, Vincent Batts wrote:
> > +* [huawei-openlab/ocp-testing](https://github.com/huawei-openlab/ocp-testing) - Testing framework for OCI configuration and runtime
> 
> Having not run it, I can't confirm. But their page says "The testing
> includes: ocp specification, container function and container
> performance."

I haven't run the tester either, but from skimming [1] it seemed like
the specs tests were “is this runtime compatible with
opencontainers/specs”, not “is this bundle-specification compatible
with opencontainers/specs”.

@liangchenye, @zenlinTechnofreak, @ChengTiesheng, @maquanyi and other
@huawei-openlab folks, can you clarify?

[1]: https://github.com/huawei-openlab/ocp-testing/tree/master/Cases/Specstest